ts/model_service_worker.py

```diff
@@ -144,6 +144,14 @@ def load_model(self, load_model_request):
             return service, "loaded model {}".format(model_name), 200
         except MemoryError:
             return None, "System out of memory", 507
+        except Exception as ex:  # pylint: disable=broad-except
+            if "CUDA" in str(ex):
+                # Handles Case A: CUDA error: CUBLAS_STATUS_NOT_INITIALIZED (Close to OOM) &
+                # Case B: CUDA out of memory (OOM)
+                return None, "System out of memory", 507
```

Comment:
Not sure how I feel about a 507 error since system out of memory doesnt typically refer to an OOM, do you know what convention other serving frameworks use?